File: pyg_sellergoods_interface_71/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;

import java.util.List;

public interface BrandService {

    public List<TbBrand> findAll();

    public PageResult<TbBrand> findPage(int page, int size);

    public void save(TbBrand tbBrand);

    public TbBrand findById(long id);

    public void delete(long[] ids);
}

File: pyg_manager_web_71/src/main/webapp/js/service/brandService.js
app.service("brandService",function ($http) {
    this.findAll = function () {
        return $http.get('../brand/findAll');
    }
    this.findPage = function (page,size) {
        return $http.get('../brand/findPage/'+page+'/'+size);
    }
    this.save = function (brand) {
        return $http.post("../brand/save", brand);
    }
    this.edit = function (id) {
        return $http.get("../brand/edit/"+id);
    }
    this.delete = function (selectIds) {
        return $http.get("../brand/delete/"+selectIds)
    }

})